R/new-model.R

```diff
@@ -90,7 +91,7 @@ new_model <- function(
 #' @seealso [copy_model_from()], [new_model()]
 #' @export
 read_model <- function(.path) {
-  yaml_path <- paste0(.path, ".yaml")
+  yaml_path <- paste0(.path %>% sanitize_file_extension(), ".yaml")
```

Comment:
Everything in here (tests, reqs, issue that prompted this) is about `new_model()`.  Is this `read_model()` change intentional?